src/domain/networking/packets/EntityData.ts

```diff
@@ -0,0 +1,40 @@
+//
+//  EntityData.ts
+//
+//  Created by Julien Merzoug on 16 May 2022.
+//  Copyright 2022 Vircadia contributors.
+//  Copyright 2022 DigiSomni LLC.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+
+type EntityDataDetails = {
+    // WEBRTC TODO: Address further code.
+};
+
+
+const EntityData = new class {
```

Comment:
The EntityData packet needs to be added to the packet list documentation in PacketHeaders.ts:
![image](https://user-images.githubusercontent.com/7455448/169449143-292cb246-e3fd-475d-8d29-15897b857b4c.png)

Reply:
Added.